robot/Cumulus/resources/NPSP.py

```diff
@@ -1588,6 +1588,8 @@ def click_special_button(self,title):
         """This keyword is similar to click button but uses set focus to button and javascript
```

Comment:
Can we update the method description with little more details?

Reply:
done, please take a look again